src/bin/sage-fixdoctests

```diff
@@ -305,17 +311,21 @@ def process_block(block, src_in_lines, file_optional_tags):
 
 
 # set input and output files
-if len(args.filename) == 2 and not args.overwrite and not args.no_overwrite:
-    inputs, outputs = [args.filename[0]], [args.filename[1]]
-    print("sage-fixdoctests: When passing two filenames, the second one is taken as an output filename; "
-          "this is deprecated. To pass two input filenames, use the option --overwrite.")
-elif args.no_overwrite:
-    inputs, outputs = args.filename, [input + ".fixed" for input in args.filename]
-else:
-    inputs = outputs = args.filename
+def output_filename(filename):
+    if len(args.filename) == 2 and not args.overwrite and not args.no_overwrite:
+        if args.filename[0] == filename:
+            print("sage-fixdoctests: When passing two filenames, the second one is taken as an output filename; "
+                  "this is deprecated. To pass two input filenames, use the option --overwrite.")
+            return args.filename[1]
```

Comment:
I get
```
$ sage -fixdoctests src/sage/rings/function_field/constructor.py output
Running "sage -t -p src/sage/rings/function_field/constructor.py output"
ERROR: file or directory not found: output

sage-runtests: Skipping 'output' because it does not have one of the recognized file name extensions
too few successful tests, not using stored timings
Skipping 'output' because it does not have one of the recognized file name extensions
No fixes made in 'src/sage/rings/function_field/constructor.py'
```
Shouldn't I get the deprecation warning?

Reply:
Thanks for testing this case. I've made the necessary fixes.